### ISSUE TITLE
Avoid using eval to parse id_charset

### ISSUE THREAD

**Issue by jvkersch** [elixir-cloud-aai/drs-filer#54]
The register objects code uses the builtin `eval` function twice to assemble a set of valid characters for object IDs, for example:

https://github.com/elixir-cloud-aai/drs-filer/blob/bf420d0b4d91948f0289159744143cf54b300463/drs_filer/ga4gh/drs/endpoints/register_objects.py#L56

The idea is that the user can specify the list of admissible ID characters using a Python expression in the configuration file:

https://github.com/elixir-cloud-aai/drs-filer/blob/bf420d0b4d91948f0289159744143cf54b300463/drs_filer/config.yaml#L69

This is unsafe since it allows executing arbitrary code (try specifying `id_charset: 'import os; os.system("rm -rf /")'` in the config file), and also somewhat unnecessary in my opinion. I would recommend just specifying the list of admissible characters directly:

```yaml
id_charset: 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789.-_~'
```

**Comment by uniqueg** [comment]
Good point. Did you try and see what happens if you do `id_charset: 'import os; os.system("rm -rf /")'`? :upside_down_face:

My reasoning was that this config is only accessible to admins who are setting up apps. But you are right, why even create a possible attach vector if it's unnecessary. I guess we could use `ast.literal_eval()`, but I think your suggestion is actually more user-friendly (nobody knows/remembers the string sets in Python).

In any case, please note that the function is now [part of FOCA](https://github.com/elixir-cloud-aai/foca/blob/940408e97b911288086b929f171ce000834e7285/foca/utils/misc.py) and could/should be imported from there. I will fix it in FOCA - and then we can just import it here.
